core/data/core.joke

```diff
@@ -3851,6 +3854,33 @@
   Defaults to true"
   {:added "1.0"})
 
+(add-doc-and-meta *verbose*
+  "Whether --verbose (or equivalent) was specified by the Joker invocation.
+
+  Note: changing this (via e.g. var-set) affects only running Joker code; it
+  does not change the value used by the Go code that implements Joker. If
+  needed, it's probably better to add a set-verbose function that changes
+  both, rather than changing Joker's internals so this variable is always checked."
+  {:added "1.0"})
+
+(add-doc-and-meta *verbosity-level*
```

Comment:
Having both `*verbose*` and `*verbosity-level*` seems redundant to me as `*verbosity-level*` provides a superset of possible options compared to `*verbose*`. Also, I am not sure these need to be exposed as vars. I would rather just have private `(verbosity-level__)` function that just returns `VerbosityLevel` value.

Reply:
> Having both `*verbose*` and `*verbosity-level*` seems redundant to me as `*verbosity-level*` provides a superset of possible options compared to `*verbose*`. Also, I am not sure these need to be exposed as vars. I would rather just have private `(verbosity-level__)` function that just returns `VerbosityLevel` value.

Sounds good. I tend to default to more flexibility, but in this case it's not really warranted and actually (per the docstring) confusing. Will update PR as you suggest.